### PR TITLE
Create Executive vote - June 7, 2023.md

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**

### CODE REVIEW

File: governance/votes/Executive vote - June 7, 2023.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+- [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement): QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN - [view here](https://gateway.pinata.cloud/ipfs/QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN).
+
+#### Relevant Smart Contract Information
+
+- **MIP21_LIQUIDATION_ORACLE:** `0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30`. 
+- **RWA_TOKEN_FAB:** `0x2B3a4c18705e99bC29b22222dA7E10b643658552`.
+- **SYMBOL:** `RWA015`.
+- **NAME:** `RWA-015`.
+- **ILK:** `RWA015-A`.
+- **RWA014:** `TBD`
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:72]: Whut? Why RWA014? This doesn't make sense as a key or a value.

File: governance/votes/Executive vote - June 7, 2023.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+- **MCD_JOIN_RWA015_A:** `TBD`.
+- **RWA015_A_URN:** `TBD`.
+- **RWA015_A_JAR:** `TBD`.
+- **RWA015_A_OUTPUT_CONDUIT:** `TBD`.
+- **RWA015_A_INPUT_CONDUIT_URN:** `TBD`.
+- **RWA015_A_INPUT_CONDUIT_JAR:** `TBD`.
+- **RWA015_A_OPERATOR:** `TBD`.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:79]: Are we expecting any of the above to be part of the executive?

File: governance/votes/Executive vote - June 7, 2023.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+Please review the following forum threads containing information about RWA015 (BlockTower Andromeda) to inform your position before voting:
+- [Initial Collateral Application](https://forum.makerdao.com/t/mip90-liquid-aaa-structured-credit-money-market-fund/18428).
+- [Steakhouse Financial Legal & Risk Assessment](https://forum.makerdao.com/t/project-andromeda-risk-legal-assessment/20969).
+- [dewiz Technical Assessment](https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974).
+
+### BlockTower RWA Vault [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmPMrvfV) the following changes to the BlockTower RWA Vault Debt Ceiling parameters will be made, if this executive proposal passes:
+- Decrease the RWA010-A Debt Ceiling by 20 million DAI from 20 million DAI to **0**.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:92]:
```suggestion
- Decrease the RWA010-A Debt Ceiling by 20 million DAI from 20 million DAI to **zero DAI**.
```

File: governance/votes/Executive vote - June 7, 2023.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+- Decrease the RWA011-A Debt Ceiling by 30 million DAI from 30 million DAI to **0**.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:93]:
```suggestion
- Decrease the RWA011-A Debt Ceiling by 30 million DAI from 30 million DAI to **zero DAI**.
```

File: governance/votes/Executive vote - June 7, 2023.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+- Increase the RWA012-A Debt Ceiling by 50 million DAI from 30 million DAI to **80 million DAI**.
+
+To facilitate these changes, the [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) for RWA010-A, RWA011-A, RWA012-A, and RWA013-A will be updated to **QmY382BPa5UQfmpTfi6KhjqQHtqq1fFFg2owBfsD2LKmYU**, the agreement details may be [viewed here](https://gateway.pinata.cloud/ipfs/QmY382BPa5UQfmpTfi6KhjqQHtqq1fFFg2owBfsD2LKmYU).
+
+### Stability Scope Defined Parameter Changes
+
+As per the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104), the following parameter changes will be made, if this executive proposal passes:
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:100]: Is it worth mentioning that all these changes are a result of the modified base rate, and potentially linking more directly to the relevant parts of the Stability Scope?

File: governance/votes/Executive vote - June 7, 2023.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+- Increase the ETH-B Stability Fee by 0.99% from 3.25% to **4.24%**.
+- Increase the ETH-C Stability Fee by 2.49% from 1% to **3.49%**.
+- Increase the WSTETH-A Stability Fee by 1.99% from 1.75% to **3.74%**.
+- Increase the WSTETH-B Stability Fee by 2.49% from 1% to **3.49%**.
+
+### Non-Scope Defined Paramater Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmQXhS3Z), the following parameter changes will be made, if this executive proposal passes:
+
+#### [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam) Increase
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:116]: Personally I'd divide these up differently to reduce the number of headers. I would likely do:

#### rETH Changes
etc

#### Other Stability Fee Changes
etc

File: governance/votes/Executive vote - June 7, 2023.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+- Increase the RETH-A Maximum Debt Ceiling by 30 million DAI from 20 million DAI to **50 million DAI**.
+
+#### [Target Availible Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Increase
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:119]:
```suggestion
#### [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Increase
```

Comment (LongForWisdom) [reply governance/votes/Executive vote - June 7, 2023.md:119]: Run through grammarly to more reliably catch this stuff

File: governance/votes/Executive vote - June 7, 2023.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+- Increase the RETH-A Stability Fee by 2.99% from 0.75% to **3.74%**.
+- Increase the CRVV1ETHSTETH-A Stability Fee by 2.49% from 1.75% to **4.24%**.
+- Increase the WBTC-A Stability Fee by 0.9% from 4.90% to **5.80%**.
+- Increase the WBTC-B Stability Fee by 1.4% from 4.90% to **6.30%**.
+- Increase the WBTC-C Stability Fee by 0.65% from 4.90% to **5.55%**.
+
+### USDP PSM [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) Reduction
+
+As per [this poll](https://vote.makerdao.com/polling/QmQYSLHH), the following change to the USDP PSM Debt Ceiling will be made, if this executive proposal passes:
+- Reduce the USDP PSM (technical name PSM-PAX-A) Debt Ceiling by 500 million DAI from 500 million DAI to **0**.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:132]:
```suggestion
- Reduce the USDP PSM (technical name PSM-PAX-A) Debt Ceiling by 500 million DAI from 500 million DAI to **zero DAI**.
```

File: governance/votes/Executive vote - June 7, 2023.md [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+
+*Technical implementation note:* to achieve this, the USDP PSM will be removed from the Autoline (DC-IAM) and have its Debt Ceiling set to 0.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:134]: Probably unnecessary inclusion imo, but not a big deal.

File: governance/votes/Executive vote - June 7, 2023.md [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+
+### MKR Vesting Transfers
+
+The following MKR vesting transfers will be made, if this executive proposal passes:
+
+#### Sidestream
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:140]:
```suggestion
#### Sidestream Core Unit
```

File: governance/votes/Executive vote - June 7, 2023.md [hunk 11]
```diff
@@ -0,0 +1,215 @@
...
+
+As per their [approved budget](https://mips.makerdao.com/mips/details/MIP40c3SP44), **348.28 MKR** will be transferred to the Sidestream Auditor Wallet at [0xb1f950a51516a697E103aaa69E152d839182f6Fe](https://etherscan.io/address/0xb1f950a51516a697E103aaa69E152d839182f6Fe).
+
+*Note:* the distributed amount has differed from the initial estimate due to Sidestream [repricing](https://forum.makerdao.com/t/mip40c3-sp44-adding-sidestream-auction-services-core-unit-mkr-budget-sas-001/10802/7) their vesting plan. It remains within the overall approved cap.
+
+#### DUX
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:146]:
```suggestion
#### DUX Core Unit
```

File: governance/votes/Executive vote - June 7, 2023.md [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+
+As per their [approved budget](https://mips.makerdao.com/mips/details/MIP40c3SP27), **225.12 MKR** will be transferred to the DUX Auditor Wallet at [0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad](https://etherscan.io/address/0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad).
+
+*Note:* the distributed amount has differed from the initial estimate due to additional hires. It remains within the overall approved cap.
+
+### Spark Protocol [Dai Direct Deposit Module](https://manual.makerdao.com/module-index/module-dai-direct-deposit) (D3M) Parameter Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmWatYqy), the following Spark Protocol D3M parameter changes will be made, if this executive proposal passes:
+- Increase the [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam) by 15 million DAI from 5 million DAI to **20 million DAI**.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:155]:
```suggestion
- Increase the [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 15 million DAI from 5 million DAI to **20 million DAI**.
```

Comment (LongForWisdom) [reply governance/votes/Executive vote - June 7, 2023.md:155]: Link direct to heading

File: governance/votes/Executive vote - June 7, 2023.md [hunk 13]
```diff
@@ -0,0 +1,215 @@
...
+- Increase the [Target Availible Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) by 15 million DAI from 5 million DAI to **20 million DAI**.
+
+### Spark Protocol Proxy (`EXEC_PROXY_SPARK`) Authorization
+
+As per [this poll](https://vote.makerdao.com/polling/Qmc9fd3j), a Spark Protocol Proxy will be authorized by this spell to allow parallel production of executive proposals covering Spark Protocol. This will allow the Spark Protocol team to prepare code separately, but any changes will not be active until triggered by a Maker Core executive proposal.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:160]: Can link to here again somewhere? <https://forum.makerdao.com/t/executive-proxies-and-spark-protocol/20976>

File: governance/votes/Executive vote - June 7, 2023.md [hunk 14]
```diff
@@ -0,0 +1,215 @@
...
+
+If passed this executive proposal will:
+- Give the Spark Proxy admin access on Spark Protocol.
+- Add the Spark Proxy to the [Emergency Shutdown Module](https://docs.makerdao.com/smart-contract-modules/shutdown/emergency-shutdown-module).
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - June 7, 2023.md:164]: Do we have an explanation of what this is or why anywhere we can reference?

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
See comments

File: governance/votes/Executive vote - June 7, 2023.md [hunk 15]
```diff
@@ -0,0 +1,215 @@
...
+- [Autoline (DC-IAM)](https://manual.makerdao.com/module-index/module-dciam): **Yes**.
+- [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam): **2.5 million DAI**.
+- [Target Availible Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap): **2.5 million DAI**.
+- [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl): **84,600 seconds (1 day)**.
+- [Liquidation Ratio (`mat`)](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): **100%**.
+- Oracle Price: **2.5 million DAI**.
+- Debt write-off timelock (`tau`): **0**.
+- Liquidation Process: Will use the established RWA collateral contracts. See [MIP21](https://mips.makerdao.com/mips/details/MIP21) for details.
+- [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement): QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN - [view here](https://gateway.pinata.cloud/ipfs/QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN).
```
[the remark this answers is not in the document]
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - June 7, 2023.md:63]: They deliberately pin the raw markdown to IPFS so I'm not sure it's possible

File: governance/votes/Executive vote - June 7, 2023.md (hunk 1 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - June 7, 2023.md:72]: Ctrl+c, Ctrl+v error, will have the address of the RWA015 token here

File: governance/votes/Executive vote - June 7, 2023.md (hunk 6 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - June 7, 2023.md:116]: changed it, see what you think

File: governance/votes/Executive vote - June 7, 2023.md (hunk 9 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - June 7, 2023.md:134]: always find this difficult to balance, try to explain the underlying technical changes for those checking the code, but does run the risk of over-complication

File: governance/votes/Executive vote - June 7, 2023.md (hunk 14 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - June 7, 2023.md:164]: No, will ask Sam for an explainer

**Review by LongForWisdom — COMMENTED** [review]
Approved pending final noted additions

**Pull request merged**